routing.rst

```diff
@@ -824,8 +824,10 @@ Parameter Conversion
 
 A common routing need is to convert the value stored in some parameter (e.g. an
 integer acting as the user ID) into another value (e.g. the object that
-represents the user). This feature is called "param converter" and is only
-available when using annotations to define routes.
+represents the user). This feature is called "param converter" and 
+configuration of this feature is only available through the 
+``@ParamConverter`` annotation. Note that the param converter itself is 
+totally indedependent of the format used to define the route itself.
```

Comment:
I think the last sentence is confusing. I did not understand the meaning of `format` on my first read.

I think you are talking about the routing configuration?
We can use the `@ParamConverter` annotation even with `@Route` annotation, yaml, etc... ?